ui/src/App.js

```diff
@@ -106,16 +107,22 @@ class App extends Component {
         return newValueList;
     }
 
-    serializeFilters(searchFilters) {
-        let filterStr = [];
-        searchFilters.forEach((values, key) => {
+  /**
+   * Converts a Map of filters to an Array of filter strings interpretable by
+   * the backend
+   * @param filterMap Map of facetName:[facetValues] pairs
+   * @return [string] Array of "facetName=facetValue" strings
+   */
+    filterMapToArray(filterMap) {
+        let filterArray = [];
+        filterMap.forEach((values, key) => {
             if (values.length > 0) {
```

Comment:
// Scenario where there are no values for a key:
// A single value is checked for a facet. The value is unchecked.
// The facet name will still be a key in filterMap, but there will be no values.
if (values.length > 0) {

Reply:
Done